src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Emitters/EndpointParameterEmitter.cs

```diff
@@ -404,8 +404,8 @@ internal static void EmitKeyedServiceParameterPreparation(this EndpointParameter
         codeWriter.EndBlock();
 
         var assigningCode = endpointParameter.IsOptional ?
-            $"httpContext.RequestServices.GetKeyedService<{endpointParameter.Type}>({endpointParameter.KeyedServiceKey});" :
-            $"httpContext.RequestServices.GetRequiredKeyedService<{endpointParameter.Type}>({endpointParameter.KeyedServiceKey})";
+            $"httpContext.RequestServices.GetKeyedService<{endpointParameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)}>({endpointParameter.KeyedServiceKey});" :
```

Comment:
```suggestion
            $"httpContext.RequestServices.GetKeyedService<{endpointParameter.Type.ToDisplayString(EmitterConstants.DisplayFormat)}>({endpointParameter.KeyedServiceKey});" :
```

We have a predefined SymbolFormat that's used elsewhere for fully-qualified types that we can use here.